test/unit/converter.test.js

```diff
@@ -92,6 +93,19 @@ describe('Converter tests', function () {
         return done();
       });
     });
+
+    it('should successfully convert a schema with circular reference', function (done) {
+      convert({ type: 'string',
+        data: circularInput
+      }, {}, function (error, result) {
+        if (error) {
+          expect.fail(null, null, error);
+          return done();
+        }
+        expect(result.result).to.be.equal(true);
```

Comment:
@umeshp7 Can we assert the request body? To make sure we get a reasonable collection generated each time